scripts/includes/common.sh

```diff
@@ -107,7 +107,11 @@ configure_st2_user () {
   sudo mkdir -p /home/stanley/.ssh
 
   # Generate ssh keys on StackStorm box and copy over public key into remote box.
-  sudo ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+  # NOTE: If the file already exists and is non-empty, then assume the key does not need
+  # to be generated again.
+  if [ ! -s /home/stanley/.ssh/stanley_rsa ]; then
+    sudo ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+  fi
```

Comment:
I think this idempotent `if` block should include more logic from the lines below, related to authorized keys and chmod.